micropython/micropython.py

```diff
@@ -64,11 +66,17 @@ def is_test_with_regex_output(test_path: str):
     return p.parent.name + "/" + p.name in TESTS_WITH_REGEX_OUTPUT
 
 
-def get_test(dut):
-    dut.expect(PROMPT, timeout=45)
-    output = dut.before
+def remove_ansi_escape(text: str):
+    ansi_escape = re.compile(r"\x1B(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~])")
```

Comment:
Maybe according to control sequence introducer pattern:

```suggestion
    ascii_escape = re.compile(r"(\x1b\[[\x30-\x3F]*[\x20-\x2F]*[\x40-\x7E])")
```

Reply:
Ok, let's leave it.

Reply:
Changed to removing only last characters